tests/backends/test_rdflib.py

```diff
@@ -5,13 +5,15 @@
 
 import pytest
 
-from tripper import Literal, Triplestore
+from tripper import RDFS, Literal, Triplestore
 
 rdflib = pytest.importorskip("rdflib")
 
 
-# Test for issue #162: Literals are lost when listing triples with rdflib
 ts = Triplestore("rdflib")
+EX = ts.bind("", "http://ex#")
+
+# Test for issue #162: Literals are lost when accessing triples with rdflib
```

Comment:
I still think this explanation is strange. Issue 162 is closed, so I do not understand. 
Maybe then just write "check that Literals are kept when accessing triples with rdflib"?
Also, put it just above the assert statement.

Reply:
This was a test added while debugging issue 162. When we have it, I think it is nice to keep the reference to the issue (it is still accessible even though it has been closed). It makes it possible to check the issue if someone would like to understand the reason for adding this test. 

The ts.parse() and assert statements belongs together. The assert statement checks what was added with the parse, so for me the placement of the comment feels logical.